packages/circuit-ui/components/ActionMenu/ActionMenu.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Status, Props, Story } from '../../../../.storybook/components';
+import * as Stories from './ActionMenu.stories';
+
+<Meta of={Stories} />
+
+# ActionMenu
+
+An ActionMenu displays a list of subsequent actions, when interacting with an actionable component.
+
+<Story of={Stories.Base} />
+<Props />
+
+- The reference element, which triggers ActionMenu, can be primary, secondary, tertiary buttons, an overflow icon, or components with embedded buttons such as the [ImageInput](Forms/ImageInput) component.
```

Comment:
"overflow icon" makes it sound like an SVG is sufficient as a trigger element (when it should be wrapped in an IconButton) and the ImageInput isn't any different from any other composite component containing a button. I would simplify this to:

```suggestion
- The reference element which triggers the ActionMenu must be a `button` element or a component that renders a `button` such as the Button or IconButton components.
```